lib/google_crawler/keywords.ex

```diff
@@ -95,4 +98,20 @@ defmodule GoogleCrawler.Keywords do
   end
 
   defp filter_by_url(keywords, _), do: keywords
+
+  defp filter_by_result_count(keywords, %{
+         "result_count_operation" => result_count_operation,
+         "result_count_value" => result_count_value
+       }) do
+    {value, _} = Integer.parse(result_count_value)
+
+    case result_count_operation do
+      ">" -> where(keywords, [k], fragment("? > ?", k.result_count, ^value))
+      "<" -> where(keywords, [k], fragment("? < ?", k.result_count, ^value))
+      "=" -> where(keywords, [k], fragment("? = ?", k.result_count, ^value))
+      _ -> keywords
+    end
+  end
+
+  defp filter_by_result_count(keywords, _), do: keywords
```

Comment:
Should we group both `filter_by_result_count` functions together @hanam1ni ?

Reply:
@andyduong1920 this function is needed, there is a case that params do not have `result_count_operation` or `result_count_value` so it not filter by this one 😁

Reply:
Oh before I saw **another function** in between these 2 functions

```
defp filter_by_result_count(keywords, %{"result_count_operation" => result_count_operation, ...}
```

and 

```
defp filter_by_result_count(keywords, _), do: keywords
```

so my comment was mean group both `defp filter_by_result_count` sit to next together, I wasn't mean `join` both functions together 😁

But so far, for now, there is no function in between these 2 functions anymore, maybe my 👓 was having some issue at that time 😀. All good now Micky, sorry for this comment 🙏